immersive-web-wg-charter.html

```diff
@@ -388,6 +388,13 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dd>
               The Immersive Web Community Group will provide the WebXR Device API seed specification to begin the standards process.  In addition, the Immersive Web Working Group plans to partner closely with the IWCG to incubate new features - in particular,  incubation of features that are out of current scope for the working group will happen in the Community Group, and then be followed by future WG rechartering to include them in scope.
             </dd>
+            <dt>
+              <a href="https://www.w3.org/community/idiw/">Inclusive Design for Immersive Web Community
```

Comment:
page says 'Inclusive Design for *the* Immersive Web'...

Reply:
well spotted, thanks!